app/models/setting/provisioning.rb

```diff
@@ -62,6 +62,7 @@ def self.default_settings
       self.set('name_generator_type', N_("Random gives unique names, MAC-based are longer but stable (and only works with bare-metal)"), 'Random-based', N_("Type of name generator"), nil, {:collection => Proc.new {NameGenerator::GENERATOR_TYPES} }),
       self.set('default_pxe_item_global', N_("Default PXE menu item in global template - 'local', 'discovery' or custom, use blank for template default"), nil, N_("Default PXE global template entry")),
       self.set('default_pxe_item_local', N_("Default PXE menu item in local template - 'local', 'local_chain_hd0' or custom, use blank for template default"), nil, N_("Default PXE local template entry")),
+      self.set('intermediate_ipxe_script', N_("Intermediate iPXE script for unattended installations"), 'intermediate', N_("Intermediate iPXE script"), nil, { :collection => Proc.new { Hash[ProvisioningTemplate.unscoped.of_kind(:iPXE).map { |tmpl| [tmpl.name, tmpl.name] }] } }),
```

Comment:
Oh I noticed this too late, but still. The default value does not correspond the template name which is "iPXE intermediate script", therefore bootstrapping always gives 404.